build.gradle

```diff
@@ -20,6 +20,12 @@ if (ext.isFips) {
 } else {
     ext.awsLcGitVersionId = 'v1.30.1'
 }
+
+// Check for user inputted git version ID.
+if (System.properties["AWSLC_REF"]) {
+    ext.awsLcGitVersionId = System.properties["AWSLC_REF"]
```

Comment:
Also, I'm pretty sure you meant to use `System.env.AWSLC_REF`. What you have here fetches from the JVM System Properties which is a totally different thing.

https://docs.gradle.org/current/userguide/build_environment.html#sec:gradle_environment_variables

Reply:
Would that still be able to access input from the gradle build command using -D (example below)? I tested the change and it didn't seem to work the same way, but I may have misunderstood. 

ex:
./gradlew build DAWSLC_REF="version"

Reply:
I reviewed the conventions in our Gradle script, it looks like we've developed a preferences for using Java system properties like this. What you have here is consistent and that's good. Bigger picture, the nice thing about using environmental variables to manipulate a build is that they are managed external to Gradle and are part of the POSIX standard.

```
# Set them within the scope of your shell
export AWSLC_REF="version"
./gradlew build

# Set them within the scope of your gradle process
AWSLC_REF="version" ./gradlew build

# Set them as part of a python script executing a process...
env = {'AWSLC_REF' : 'version'}
subprocess.run(['gradlew', 'build'], env=env)

```

Reply:
In retrospect, we should have used [Gradles properties](https://docs.gradle.org/current/userguide/project_properties.html) instead of JVM properties or even environment variables.

I'm not suggesting changing that in this PR since the use of JVM properties has been preferred.